app-modules/audit/database/migrations/2023_08_16_025823_create_audit_settings.php

```diff
@@ -6,7 +6,7 @@
 return new class () extends SettingsMigration {
     public function up(): void
     {
-        $this->migrator->add('audit.retention_duration', 129600);
+        $this->migrator->add('audit.retention_duration', 90);
```

Comment:
I'm wondering if we also need to define a unit, or make this setting more explicit in the code that it's in days.

Reply:
Hmm, like perhaps a comment within the settings class for this?

Reply:
Fixed, went with option #1 for now